distro/wildfly20/src/main/resources/overlay/standalone/data/h2/keycloak.trace.db

```diff
@@ -0,0 +1,75 @@
+2020-08-13 11:53:33 jdbc[3]: exception
```

Comment:
You should likely remove this

Reply:
Don't think you need keycloak.mv.db either

Reply:
Hmm, might be wrong on that latter one? Double check 😂

Reply:
I probably don't need the .trace.db file - but the .mv.db file is the keycloak database with all of the proper settings and everything already configured.  So that one is definitely needed.  H2 has moved from *.h2.db files to *.mv.db files.  Some sort of new table or file format I guess.